wao/scalping_strategy_tester.py

```diff
@@ -37,6 +37,12 @@ def get_year_from_timestamp(unix_time) -> str:
     return str(date.year)
 
 
+def get_win_rate_percentage_per_year() -> str:
+    out_put_to_be_parsed = run_scalping_strategy_command()
```

Comment:
This is double computing , we shouldn't run this command again here just for the name.  That's why I recommended to add it in the file instead of the name 


This method needs to only be called once and it's already called 

Use result from where this is called and add in file

Reply:
how about now